goal_src/jak2/kernel/gkernel-h.gc

```diff
@@ -763,3 +763,11 @@
 (defmacro current-time ()
   `(-> PP clock frame-counter)
   )
+
+(defmacro seconds-per-frame ()
+  "Macro for assuming a 16.6 ms frame time at higher frame rates."
+  `(if (= (get-video-mode) 'custom)
+     0.016666668
+     (-> PP clock seconds-per-frame)
```

Comment:
This seems kinda weird, why would we set the frame time to 16.6 ms when running at a higher frame rate?

Reply:
I just adapted it from this particular patch: https://github.com/open-goal/jak-project/blob/master/goal_src/jak1/engine/target/target.gc#L813

I wasn't quite sure if this is the right move, but I did find another case where it fixed an issue (the jetboard spin).